Source/WTF/wtf/WTFConfig.cpp

```diff
@@ -114,6 +114,10 @@ void Config::initialize()
     }();
     g_wtfConfig.highestAccessibleAddress = static_cast<uintptr_t>((1ULL << OS_CONSTANT(EFFECTIVE_ADDRESS_WIDTH)) - 1);
     SignalHandlers::initialize();
+
+    uint8_t* reservedConfigBytes = reinterpret_cast_ptr<uint8_t*>(WebConfig::g_config + WebConfig::reservedSlotsForExecutableAllocator);
+    const char* useAllocationProfiling = getenv("JSC_useAllocationProfiling");
+    reservedConfigBytes[WebConfig::ReservedByteForAllocationProfiling] = (useAllocationProfiling && atoi(useAllocationProfiling)) ? 1 : 0;
```

Comment:
I would do `equalLettersIgnoringASCIICase(span, "true"_s) || equalLettersIgnoringASCIICase(span, "yes"_s) || !strcmp(string, "1")` instead of `atoi`

Reply:
Adopted this (with a null check before we construct the span) in the latest push.